internal/admission/validator.go

```diff
@@ -298,6 +298,10 @@ func (validator KongHTTPValidator) ValidatePlugin(
 	}
 	if k8sPlugin.ConfigFrom != nil {
 		if len(plugin.Config) > 0 {
+			// TODO: remove when KIC supports only Kubernetes 1.25+ and this no
+			// longer needs to be done in admission webhook.
+			// CRD validation expression is already in place since
+			// https://github.com/Kong/kubernetes-ingress-controller/pull/5119
```

Comment:
I think we could already remove that given we decided to claim only Kubernetes 1.25+ support for KIC 3.0.

Reply:
I misinterpreted our last discussion on slack which stated that we support 1.24 as in "KIC runs on 1.24, possibly with impaired validation" 😅 Let me remove this note and this branch in here.

Reply:
Changed PTAL.

Reply:
I also have the same question, but after seeing the discussion here, my question has already been resolved.